tests/ops/qubit/test_parametric_ops.py

```diff
@@ -786,6 +792,50 @@ def test_controlled_phase_shift_decomp(self, cphase_op):
 
         assert np.allclose(decomposed_matrix, exp)
 
+    @pytest.mark.parametrize("phi", [-0.1, 0.2, 0.5])
+    @pytest.mark.parametrize(
+        "cphase_op,lam_pos",
+        [
+            (qml.CPhaseShift00, 0),
+            (qml.CPhaseShift01, 1),
+            (qml.CPhaseShift10, 2),
+        ],
+    )
+    def test_c_phase_shift_decomp(self, phi, cphase_op, lam_pos):
+        """Tests that the CPhaseShift operations
+        calculate the correct decomposition"""
+        op = cphase_op(phi, wires=[0, 2])
+        decomposed_matrix = qml.matrix(op.decomposition, wire_order=op.wires)()
+        lam = np.exp(1j * phi)
+        exp = np.eye(4, dtype=complex)
+        exp[..., lam_pos, lam_pos] = lam
+
+        assert np.allclose(decomposed_matrix, exp)
+
+    @pytest.mark.parametrize(
+        "cphase_op,lam_pos",
+        [
+            (qml.CPhaseShift00, 0),
+            (qml.CPhaseShift01, 1),
+            (qml.CPhaseShift10, 2),
+        ],
+    )
+    def test_c_phase_shift_decomp_broadcasted(self, cphase_op, lam_pos):
+        """Tests that the CPhaseShift operations
+        calculate the correct decomposition"""
+        phi = np.array([-0.2, 4.2, 1.8])
+        op = cphase_op(phi, wires=[0, 2])
+        decomposed_matrix = qml.matrix(op.decomposition, wire_order=op.wires)()
+        lam = np.exp(1j * phi)
+        exp = []
+        for el in lam:
+            exp_el = np.eye(4, dtype=complex)
+            exp_el[..., lam_pos, lam_pos] = el
+            exp.append(exp_el)
+        exp = np.array(exp)
+
+        assert np.allclose(decomposed_matrix, exp)
```

Comment:
The problem with this test seems to be with the `qml.matrix` function rather than the decomposition.  

I normally am a fan of over testing everything, but in this case, I don't think we really need to test the broadcasted decomposition.  Tests already exist for broadcasted matrix and for normal decomposition.  

I would just delete this test.
```suggestion
```